### ISSUE TITLE
occa-dev branch compile error

### ISSUE THREAD

**Issue by BamboWu** [CEED/Laghos#16]
I want to run Laghos on NVIDIA P100, so I am trying to compile the occa-dev branch.
However, the compiler complains at declarations with occa involved.
For example: 
~~~bash
laghos_assembly.hpp:34:7: error: 'occa' does not name a type
       occa::device device;
       ^
~~~
Is it because my makefile is not well configured?
I am looking forward to get some instructions about how to compile the occa-dev branch appropriately.
Thanks in advance!

**Comment by v-dobrev** [comment]
Hello @baby-panda,

I believe @dmed256 is the best person to provide some build instructions for the `occa-dev` branch.

@dmed256, do you mind answering @baby-panda's question? Thanks!

**Comment by dmed256** [comment]
@baby-panda We just released OCCA 1.0 and although the MFEM+OCCA branch seems to be ported over, I'm still finishing porting the Laghos+OCCA branch right now. I'll update this issue once I can get it running again

### Installation steps
**OCCA**
```
git clone https://github.com/libocca/occa.git
cd occa
make
export OCCA_DIR=$PWD 
```

> Note: MFEM uses OCCA_DIR to find where OCCA is installed

**MFEM**
```
git clone https://github.com/mfem/mfem.git
cd mfem
git checkout occa-dev
make config MFEM_USE_OCCA=YES MFEM_USE_MPI=YES
make
```
**Laghos**
```
git clone https://github.com/CEED/Laghos.git
cd Laghos
git checkout occa-dev
make
```

**Comment by BamboWu** [comment]
Thanks a lot!
I will follow your update.
